server/src/rest_service.py

```diff
@@ -63,7 +63,7 @@ def add_team(db_session, data):
         return True, "Successfully added Team {}".format(data["name"])
     except Exception as e:
         db_session.rollback()
-        return False, "Could not add Team {}; already exists".format(data["name"])
+        return False, "Could not add Team {}; already exists".format(e)
```

Comment:
I thought that I should revert this change but thinking about it now I think I should keep it. The e gives better contextual feedback since its not always an already exists error.